javagen/src/main/java/com/azure/autorest/model/clientmodel/MethodDocumentation.java

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.autorest.model.clientmodel;
+
+import java.util.Objects;
+
+public class MethodDocumentation {
+    private String description;
+    private String url;
+
+    protected MethodDocumentation(String description, String url) {
+        this.description = description;
+        this.url = url;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public String getUrl() {
+        return url;
+    }
+
+
+    @Override
+    public String toString() {
+        return "MethodDocumentation{" +
+                "description='" + description + '\'' +
+                ", url='" + url + '\'' +
+                '}';
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        MethodDocumentation that = (MethodDocumentation) o;
+        return Objects.equals(description, that.description) && Objects.equals(url, that.url);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(description, url);
+    }
```

Comment:
Do we need to override these? It's good but curious about where we'd need this.

Reply:
We don't use these two methods currently, I will delete it.